deploy/apex/overlays/cockroach/secrets.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```

Comment:
I would use a `secretGenerator` in `kustomization.yaml` for these:
https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kustomize/

Reply:
there are actually no secrets here.  this dev deployment of cockroach accepts ANY password.